libs/net/dal/Migrations/TNOContextModelSnapshot.cs

```diff
@@ -567,6 +567,98 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("action");
                 });
 
+            modelBuilder.Entity("TNO.Entities.CBRAReportStaffSummary", b =>
```

Comment:
Are you manually adding these?  I'll need to test what happens after this migration is applied to determine whether this will impact future migrations.